ddtrace/_monkey.py

```diff
@@ -222,8 +224,8 @@ def patch(raise_errors=True, patch_modules_prefix=DEFAULT_MODULES_PREFIX, **patc
 
         >>> patch(psycopg=True, elasticsearch=True)
     """
-    contribs = [c for c, should_patch in patch_modules.items() if should_patch]
-    for contrib in contribs:
+    contribs = {c: should_patch for c, should_patch in patch_modules.items() if should_patch}
```

Comment:
does the typing of this function need to change? does it need to now be: `(bool, str, Union[List[str], bool]) -> None` ?

Reply:
That is correct. Will be updated in the next commit.